src/Util/InheritanceHelper.php

```diff
@@ -10,60 +11,57 @@
 
 class InheritanceHelper
 {
-    /**
-     * @param string $methodName
-     * @param Tokens $tokens
-     * @return bool
-     */
-    public function isMethodFromInterface($methodName, Tokens $tokens)
+    public function isMethodFromInterface(string $methodName, Tokens $tokens): bool
     {
         try {
             $reflection = $this->getReflection($tokens);
         } catch (ReflectionException $exception) {
             return false;
         }
 
-        foreach ($reflection->getInterfaces() as $interface) {
-            $methods = $interface->getMethods(ReflectionProperty::IS_PUBLIC | ReflectionProperty::IS_PROTECTED);
-            foreach ($methods as $method) {
-                if ($method->getName() === $methodName) {
-                    return true;
+        if ($reflection !== null) {
```

Comment:
Good opportunity to use guard clause
```
if ($reflection === null) {
    return false;
}
// ...
```
In other places too.

Reply:
Fixed